backend/app/routes/events.py

```
@@ -44,169 +46,137 @@ def get_language_from_header(accept_language: Optional[str] = Header(None)) -> s
     return DEFAULT_LANGUAGE
 
 
+# Safe wrapper functions for EventsService operations
+def _safe_search_events(search_params: EventSearchParams, accept_language: Optional[str] = None):
+    """Safe wrapper for EventsService.search_events with retry logic."""
+    def _search_operation(db: Session):
+        from ..core.translation import get_translation_service
```

Ruff (0.12.2)

Redefinition of unused `get_translation_service` from line 19

Remove definition: `get_translation_service`

(F811)